lib/web_of_science/harvester.rb

```diff
@@ -10,13 +10,19 @@ class Harvester < ::Harvester::Base
     # @param [Enumerable<Author>] authors
     # @return [void]
     def harvest(authors)
+      logger.info "#{self.class} - started harvest(authors) - #{authors.count} authors"
+      author_success = 0
       authors.each do |author|
         begin
           process_author(author)
-        rescue StandardError => e
-          NotificationManager.error(e, "WebOfScience harvest failed for author: '#{author.id}'", self)
+          logger.info "#{self.class} - processed author: #{author.id}"
```

Comment:
If the problem is identifying a given author that blows things up, you probably want to log *before* `process_author`.  Otherwise the log will only ever identify the author previous to the problematic one.

Reply:
(Because OOM kills the process and no `StandardError` is caught.)

Reply:
Pushed up a revision, placed the author logging in the process_author method instead.